docker/datahub-gms/start.sh

```diff
@@ -23,4 +23,7 @@ dockerize \
   -wait $ELASTICSEARCH_PROTOCOL://$ELASTICSEARCH_HOST_URL:$ELASTICSEARCH_PORT \
   -wait $NEO4J_HOST \
   -timeout 240s \
-  java -jar /jetty-runner.jar /datahub/datahub-gms/bin/war.war
\ No newline at end of file
+  java $JAVA_OPTS $JMX_OPTS \
+  --jar jetty-util.jar \
```

Comment:
@bernardino This change is breaking prod. I think java only takes a single jar. Has this specific script been tested locally?

Reply:
I think I know what the error is, it should be
```
java $JAVA_OPTS $JMX_OPTS \
-jar /jetty-runner.jar \
--jar jetty-util.jar \
--jar jetty-jmx.jar \
/datahub/datahub-gms/bin/war.war
```

The `--jar` I added is for the jetty runner, it has nothing to do with the `-jar`, and in our custom run script in our helm charts the `--jar` options are after the `runner.jar`